scripts/makefile_helper

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2022-present Team LibreELEC (https://libreelec.tv)
+
+# default variables
+CLEAN="false"
+DISTCLEAN="false"
+ROOT="${PWD}"
+
+
+# Just interested in how the builder set BUILD_DIR...
+if [ -f "${ROOT}/.libreelec/options" ]; then
+  . "${ROOT}/.libreelec/options"
+fi
+
+if [ -f "${HOME}/.libreelec/options" ]; then
+  . "${HOME}/.libreelec/options"
+fi
+
+BUILD_DIR="${BUILD_DIR:-${ROOT}}"
+
+
+# CLI argument handling
+case $1 in
+  -c|--clean)
+  CLEAN="true"
+  ;;
+  -d|--distclean)
+  DISTCLEAN="true"
+  ;;
+  *)
+  echo "error: ${0}: unsupported option on CLI; aborting"
+  exit 1
+  ;;
+esac
+
+
+# cleanup tasks
+if [ ${CLEAN} = "true" ]; then
+  rm -rf ${BUILD_DIR}/build.*/* ${BUILD_DIR}/build.*/.stamps
```

Comment:
```suggestion
  rm -rf "${BUILD_DIR}"/build.*/* "${BUILD_DIR}"/build.*/.stamps
```

not 100% sure what the bash gurus think about it ?

Reply:
This would be required if we permitted people to use a build location with a space in it. I've incorporated anyway.